test/e2e/codecept.conf.js

```diff
@@ -9,7 +9,7 @@ exports.config = {
     Nightmare: {
       url: process.env.TEST_URL || config.get('e2e.frontendUrl'),
       waitForTimeout: parseInt(config.get('e2e.waitForTimeout')),
-      waitForAction: parseInt(config.get('e2e.waitForAction')),
+      waitForAction: parseInt(process.env.WAIT_FOR_ACTION || config.get('e2e.waitForAction')),
```

Comment:
Does E2E_WAIT_FOR_ACTION_VALUE not work as that has been set in custom-environment-variables.json. No idea if this works for tests though?

Reply:
Thanks @chrisgrimble. I'll take a look as I didn't know about that.